spec/Powertrain/Battery.vspec

```diff
@@ -83,3 +84,103 @@
   unit: kWh
   description: The accumulated energy leaving HV battery for propulsion and auxiliary loads over lifetime.
 
+
+- Charging:
+  type: branch
+  description: Properties related to battery charging
+  
+- Charging.ChargeLimit:
+  datatype: UInt8
+  type: actuator
+  min: 0
+  max: 100
+  unit: percent
+  description: Maximum charge level for battery, can potentially be set manually.
+
+- Charging.MaximumChargingCurrent:
+  datatype: float
+  type: sensor
+  unit: A
+  description: Maximum charging current that can be accepted by the system.
+
+- Charging.ChargePortFlap:
+  datatype: string
+  type: actuator
+  value: closed
+  enum: [ 'open', 'closed' ]
+  description: Signal indicating if charge port cover is open or closed, can potentially be controlled manually.
+
+- Charging.ChargePlugStatus:
+  datatype: boolean
+  type: sensor
+  description: Signal indicating if charge plug is connected or not.
```

Comment:
Gibt es einen Unterschied zwischen "Connected" and "Locked"? Ich meine du kannst den Port ja unlocken, aber das Kabel ist noch drin....